sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -845,6 +847,16 @@ class SQLContext private[sql](
   }
 
   /**
+   * Returns a [[ContinuousQueryManager]] that allows managing all the
+   * [[org.apache.spark.sql.ContinuousQuery ContinuousQueries]] active on `this` context.
+   *
+   * @since 2.0.0
+   */
+  def streams(): ContinuousQueryManager = {
```

Comment:
I explicitly added it to keep it consistent with `tableNames()` and `tables()` is the surrounding code. I personally prefer think that none of these should have `()`